packages/react-tabs/src/components/TabList/useTabListStyles.ts

```diff
@@ -1,7 +1,14 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { TabListState } from './TabList.types';
+import { tokens } from '@fluentui/react-theme';
+import { usePrevious } from '@fluentui/react-utilities';
+import { tabPendingDesignTokens } from '../../tab.constants';
 
 export const tabListClassName = 'fui-TabList';
+export const tabListSelectionIndicatorName = 'fui-TabList_SelectionIndicator';
+
+export const indicatorOffsetVar = '--selection-indicator-offset';
```

Comment:
these should use your prefix to ensure that there aren't any possible conflicts.

```suggestion
export const indicatorOffsetVar = '--fui-TabList-selection-indicator-offset';
```

Reply:
I don't think css vars need the prefix - they are setting the value within the component and aren't meant to be addressable like the style names.  We could ask for people's thoughts at tech sync.